user/urls.py

```diff
@@ -0,0 +1,9 @@
+from django.contrib import admin
+from django.urls import path
+
+from user import views
+
+urlpatterns = [
+    path('', views.BookView.as_view(), name="form"),
+    path('index/', views.show,name="index"),
```

Comment:
instead of form write proper name as per functionality.

Reply:
views.index